src/controllers/Lane.js

```diff
@@ -26,7 +26,7 @@ class Lane extends Component {
     const node = evt.target
     const elemScrollPosition = node.scrollHeight - node.scrollTop - node.clientHeight
     const {onLaneScroll} = this.props
-    if (elemScrollPosition <= 0 && onLaneScroll && !this.state.loading) {
+    if (elemScrollPosition <= 1 && onLaneScroll && !this.state.loading) {
```

Comment:
Would `< 1` also work here?

Reply:
Yes, that'd work too. Normally the difference is never going to be 1 or more.

Reply:
Hi guys! It's very cool that you fixes that issue! 

I believe it's better to describe this line and put a link to the issue. Because if you do not know the reason, then this code looks strange.

Also I prefer `< 1` )